hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/HandlerUtility.java

```diff
@@ -259,13 +260,16 @@ static TransactionID transactionIdForScheduled(@NonNull Schedule valueInState) {
         // original create transaction and its transaction ID will never be null, but Sonar...
         final TransactionBody originalTransaction = valueInState.originalCreateTransactionOrThrow();
         final TransactionID parentTransactionId = originalTransaction.transactionIDOrThrow();
-        // payer on parent transaction ID will also never be null...
-        final AccountID payerAccount = valueInState.payerAccountIdOrElse(parentTransactionId.accountIDOrThrow());
-        // Scheduled transaction ID is the same as its parent except
-        //     if scheduled is set true, payer *might* be modified, and the nonce is incremented.
-        final TransactionID.Builder builder = TransactionID.newBuilder().accountID(payerAccount);
-        builder.transactionValidStart(parentTransactionId.transactionValidStart());
-        builder.scheduled(true).nonce(parentTransactionId.nonce() + 1);
```

Comment:
nouce setting is removed here. Don't you need that for parent/child transaction?

Reply:
No.  Workflow sets those values and mostly ignores what I set here.
This Transaction ID needs to be _exactly_ the same as the original Create transaction except for the "scheduled" flag (many tests depend on that exact match).